code/game/objects/structures/morgue.dm

```diff
@@ -151,6 +151,10 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 	connected.connected = src
 	..()
 
+/obj/structure/bodycontainer/morgue/examine(mob/user)
+	..()
+	to_chat(user, "<span class='notice'>You can Alt-click the tray to deactivate the alarm.</span>")

```

Comment:
to_chat(user, "<span class='notice'>Alt-click [src] to [beeper ? "disable" : "enable"] the alarm.</span>")